akka-http-core/src/test/scala/akka/http/javadsl/JavaInitializationSpec.scala

```diff
@@ -12,7 +12,7 @@ class JavaInitializationSpec extends WordSpec with Matchers {
 
     "initializes the right field" in {
       akka.http.scaladsl.model.headers.LanguageRange.`*` // first we touch the scala one, it should force init the Java one
-      akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
+      val _ = akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
```

Comment:
OK, I've looked into this and managed to reproduce a NPE with this patch -https://gist.github.com/jonas/6ccec6a28a105c89e3793b9f410b71ef
based on the test case provided here https://github.com/akka/akka/issues/20403#issuecomment-215356663.

I will create a ticket with these findings.